rules-reviewed/camel3/camel2/xml-renamed-components.windup.xml

```diff
@@ -79,5 +79,185 @@
                 </hint>
             </perform>
         </rule>
+        <rule id="xml-renamed-components-00004">
+            <when>
+                <project>
+                    <artifact groupId="org.apache.camel" artifactId="camel-hdfs2" />
+                </project>
+            </when>
+            <perform>
+                <hint title="`org.apache.camel:camel-hdfs2` artifact has been renamed to `org.apache.camel:camel-hdfs`" effort="1" category-id="mandatory" >
```

Comment:
For consistency with the above rules, please change the hint's title in the new rules like in this example:
```suggestion
                <hint title="`org.apache.camel:camel-hdfs2` artifact has been renamed" effort="1" category-id="mandatory" >
```